cmd/movefunds/main.go

```diff
@@ -187,7 +187,8 @@ func main() {
 	}
 
 	txOuts := []*wire.TxOut{wire.NewTxOut(int64(targetAmount-fee), pkScript)}
-	atx, err := txauthor.NewUnsignedTransaction(txOuts, fee, inputSource, nil)
+	atx, err := txauthor.NewUnsignedTransaction(txOuts, fee, inputSource, nil,
+		params.MaxTxSize)
```

Comment:
movefunds is its own module (this was done so we don't have to keep it always updated, feel free to not change this here).  If it is updated, the go.mod will need to require the new modules and there will likely be other breaking changes to fix.